integration-tests/bats/sql-server.bats

```diff
@@ -1739,3 +1739,53 @@ s.close()
     run grep "failed to access 'mydb2' database: can no longer find .dolt dir on disk" server_log.txt
     [ "${#lines[@]}" -eq 1 ]
 }
+
+@test "sql-server: dropping database that the server is running in" {
+    skiponwindows "Missing dependencies"
+
+    mkdir mydb
+    cd mydb
+    dolt init
+
+    start_sql_server >> server_log.txt 2>&1
+
+    server_query "" 1 dolt "" "DROP DATABASE mydb"
+
+    run grep "database not found: mydb" server_log.txt
```

Comment:
Running a query like `show databases;` and/or using the database to run a sanity test query would probably be a little bit more direct way to test that the toBeDeleted database is gone and that the other nested databases are still present.